tests/integration/functional/test_client.py

```diff
@@ -1323,6 +1323,26 @@ def test_latest_version_retrieval(self, clean_client: "Client"):
         assert model_.latest_version_name == mv2.name
         assert model_.latest_version_id == mv2.id
 
+    def test_list_by_tags(self, clean_client: "Client"):
```

Comment:
maybe add a test for the empty list passed in etc?

```python
ms = clean_client.list_models(tags=[])
# Expected behavior for no tags scenario

ms = clean_client.list_models(tags=[""])
# Expected behavior for empty string tag

ms = clean_client.list_models(tags=["non_existent_tag"])
assert len(ms) == 0
```

Reply:
and the same below for the others?

Reply:
Thanks, caught one bug 👍🏼